http/headers/feature-policy.json

```diff
@@ -385,23 +330,11 @@
               "edge": {
                 "version_added": "94"
               },
-              "firefox": [
-                {
-                  "version_added": "74",
-                  "partial_implementation": true,
-                  "notes": "Only supported through the <code>allow</code> attribute on <code>&lt;iframe&gt;</code> elements."
-                },
-                {
-                  "version_added": "67",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "dom.security.featurePolicy.header.enabled",
-                      "value_to_set": "true"
-                    }
-                  ]
-                }
-              ],
+              "firefox": {
+                "version_added": "74",
+                "partial_implementation": true,
+                "notes": "Only supported through the <code>allow</code> attribute on <code>&lt;iframe&gt;</code> elements."
+              },
               "firefox_android": {
                 "version_added": false
```

Comment:
No Android?

Reply:
Seems that none of the mobile browsers are marked as supported.  I guess that makes sense.